file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,588 @@
+      "! Assign result process to the UI group
+      result_process   TYPE ty_result_processes,
+    END OF ty_inconsistency_category.
+  "! <p class="shorttext">Inconsistency Categories</p>
+  "! Inconsistency categories
+  TYPES ty_inconsistency_categories TYPE SORTED TABLE OF ty_inconsistency_category WITH UNIQUE KEY result_ui_group.
+  "! <p class="shorttext">Field Type</p>
+  "! Field type
+  "! $values {@link zif_aff_edcc_v1.data:co_field_type}
+  "! $default {@link zif_aff_edcc_v1.data:co_field_type.select_options}
+  TYPES ty_field_type               TYPE c LENGTH 1.
+  TYPES:
+    "! <p class="shorttext">Additional Selection Field</p>
+    "! Additional selection fields of validation report
+    BEGIN OF ty_additional_selection_field,
+      "! <p class="shorttext">Table Name</p>
+      "! Table name
+      table_name TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Field Name</p>
+      "! Field name
+      field_name TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Field Type</p>
+      "! Field type
+      field_type TYPE ty_field_type,
+    END OF  ty_additional_selection_field.
+  "! <p class="shorttext">Additional Selection Fields</p>
+  "! Additional selection fields of validation report
+  TYPES ty_additional_selection_fields TYPE SORTED TABLE OF ty_additional_selection_field WITH UNIQUE KEY
+        table_name field_name.
+  TYPES:
+    "! <p class="shorttext">Consistency Scenario</p>
+    "! Consistency scenario
+    BEGIN OF ty_main,
+      "! $required
+      format_version              TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                      TYPE zif_aff_types_v1=>ty_header_60_cloud,
+      "! <p class="shorttext">Country</p>
+      "! Country
+      "! $required
+      country                     TYPE c LENGTH 3,
+      "! <p class="shorttext">Additional Selection Fields</p>
+      "! Additional selection fields of validation report
+      additional_selection_fields TYPE ty_additional_selection_fields,
+      "! <p class="shorttext">Tax Authority Message Types</p>
+      "! Tax authority message types
+      tax_authority_messages      TYPE ty_messages,
+      "! <p class="shorttext">Additional Tax Authority Tables</p>
+      "! Additional tax authority tables
+      tax_authority_tables        TYPE ty_tax_authority_tables,
+      "! <p class="shorttext">Comparison Types</p>
+      "! Define comparison types
+      "! $required
+      comparison_types            TYPE ty_comparison_types,
+      "! <p class="shorttext">Events Assignment</p>
+      "! Assign events to consistency scenario
+      events                      TYPE ty_events,
+      "! <p class="shorttext">Representation Types</p>
+      "! Assign representation types associated with consistency scenario
+      "! $required
+      relationship_attribute      TYPE ty_relationship_attributes,
```

Comment:
Title and name of component shall be in sync.  
However, there is already a subcomponent Representation Types in Content Checks. Do we need them twice?